browser/html/cool-help.html

```diff
@@ -6,573 +6,768 @@
     <ul>
       <li><p>Search backwards</p></li>
       <li><p>Search forward</p></li>
       <li><p>Cancel the search (appears only when a text has been searched)</p></li>
     </ul>
+  </div>
+  <div class="sub-section">
     <p><span class="def">The zoom bar:</span> At the right of the status bar, there is a set of buttons that allow you to zoom to 100%, zoom out and zoom in. The zoom applies to the document area, the user interface is not affected. The current level of zoom is shown in this area.</p>
     <p>Text, charts, shapes and svg images will stay sharp, when zooming in, you will only see pixels appear at inserted images, such as jpg’s or png’s.</p>
     <p>Using the browser zoom affects the document and user interface areas.</p>
+  </div>
+  <div class="sub-section">
     <p><span class="def">The information bar:</span> At the right of the search bar, a set of information on the document is displayed. The actual information depends on the nature of the document.</p>
     <div class="screenshot"><img alt="" src="images/help/en/information-bar.png"/></div>
+  </div>
+</div>
     <a id="1335"/>
-    <h3>Opening, closing, saving, printing and downloading documents</h3>
+    <div class="section">
+    <h3 class="section-header">Opening, closing, saving, printing and downloading documents</h3>
     <p>Your documents are stored and managed in the cloud storage that is integrated with <span class="productname">%productName</span>.</p>
-    <p>To download a document download it from the <span class="productname">%productName</span> application’s <span class="ui">File</span> menu. The download formats available depends on the application. All applications exports documents in PDF format.</p>
-    <p>To open document, click on the file to open the <span class="productname">%productName</span> module associated to the document format.</p>
-    <p>Documents in <span class="productname">%productName</span> save automatically, but if you are concerned that a document is synchronized as quickly as possible you can also force saving using the <span class="ui">File</span> menu’s <span class="ui">Save</span> entry.</p>
-    <p>Depending on the capabilities of your browser the print window may appear or a “Download PDF export?” popup shows. You can print this PDF in your favorite PDF reader.</p>
-    <p>When closing a document, it is automatically saved if it has been changed.</p>
+    <div class="sub-section"><p><span class="def">Download:</span>To download a document download it from the <span class="productname">%productName</span> application’s <span class="ui">File</span> menu. The download formats available depends on the application. All applications exports documents in PDF format.</p></div>
+    <div class="sub-section"><p><span class="def">Open:</span>To open document, click on the file to open the <span class="productname">%productName</span> module associated to the document format.</p></div>
```

Comment:
new string inside ("Open:") will  be old content still translated with this change?